src/Fluid/Flu_FixUp_Restrict.cpp

```diff
@@ -343,6 +343,24 @@ void Flu_FixUp_Restrict( const int FaLv, const int SonFluSg, const int FaFluSg,
                                         amr->patch[FaFluSg][FaLv][FaPID]->fluid[ENGY][k][j][i],
                                         MIN_EINT, Emag );
 #        endif // #ifdef DUAL_ENERGY ... else ...
+
+//       fix temperature initial guess
+#        ifdef TEMP_IG
+         real Passive[NCOMP_PASSIVE];
+
+         for (int N_PASSIVE=0; N_PASSIVE<NCOMP_PASSIVE; N_PASSIVE++)
```

Comment:
Do not use all uppercase letters for a variable name. Just use `v`.

Reply:
I have replaced it to `v`.